types/serverless/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for serverless 1.78
+// Type definitions for serverless 2.32
```

Comment:
```suggestion
// Type definitions for serverless 1.78
```

I think for now let's keep this as is, just because it'll trigger a new major version of the package which will make it seem like a lot bigger change than it is.

Ideally we should go through the types and check if we've got anything from v1 that was removed in v2, and then bump the major version.

Reply:
Yeah I wasn't sure about this either, thanks for reviewing :)